addon/index.js

```diff
@@ -1,11 +1,12 @@
 import { TrackedMap } from 'tracked-maps-and-sets';
 
+const keyCache = new Set();
```

Comment:
Do you mind renaming it to `managedKeys` or something like this? It's not actually a cache but a list of keys that are cached  and observed by `@localStorage` decorator.